harvester/database/models.py

```diff
@@ -0,0 +1,58 @@
+from sqlalchemy import text, String, Integer,  ARRAY, ForeignKey, DateTime
+from sqlalchemy.orm import DeclarativeBase, relationship, mapped_column
+from sqlalchemy.dialects.postgresql import UUID
+
+
+class Base(DeclarativeBase):
+    id = mapped_column(
+        UUID(as_uuid=True), primary_key=True, server_default=text("gen_random_uuid()")
+    )
+
+class HarvestSource(Base):
+    __tablename__ = 'harvest_source'
+    
+    name = mapped_column(String, nullable=False)
+    notification_emails = mapped_column(ARRAY(String))
+    organization_name = mapped_column(String)
+    frequency = mapped_column(String, nullable=False)
+    url = mapped_column(String, nullable=False)
+    schema_type = mapped_column(String, nullable=False)
+    source_type = mapped_column(String, nullable=False)
+    harvest_source_id = mapped_column(String)
+    harvest_source_name = mapped_column(String)
```

Comment:
what is difference between `name` & `harvest_source_name`?

Reply:
good point. I think they are the same thing.  We added the `harvest_source_id` and `harvest_source_name` which are from ckan database.  We should only keep one. Will do the modification later along with other database changes.